cmd/blockchaincmd/create.go

```diff
@@ -375,6 +389,11 @@ func createBlockchainConfig(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func getValidatorContractManagerAddr() ([]string, bool, error) {
+	// TODO: replace this with implementation in validator prompt PR
+	return nil, false, nil
```

Comment:
implement this giving user key options besides custom. also, I believe you only need one address, so, lets
return string not []string